src/order/atoms.lean

```diff
@@ -54,11 +54,11 @@ section atoms
 
 section is_atom
 
-variables [partial_order α] [order_bot α] {a b x : α}
-
 /-- An atom of an `order_bot` is an element with no other element between it and `⊥`,
   which is not `⊥`. -/
-def is_atom (a : α) : Prop := a ≠ ⊥ ∧ (∀ b, b < a → b = ⊥)
+def is_atom [preorder α] [order_bot α] (a : α) : Prop := a ≠ ⊥ ∧ (∀ b, b < a → b = ⊥)
```

Comment:
This definition no longer really makes sense. I think it should be something like `⊥ < a ∧ (∀ b, b < a → b = ⊥)`.

Reply:
I agree, but this PR is big enough that I'm keeping this for the next.

Reply:
Also, this could be stated in terms of `is_min` (from #11408): `¬ is_min a ∧ ∀ ⦃b⦄, b < a → is_min b`. That would actually allow us to use `is_atom` as the definition of `order.ideal.maximal`.